buddyboss/buddyboss-platform
# PROD-800 #2512  - Fix Document description doesn't carry over from group feed

### Changes proposed in this Pull Request:



Fixes #2512 .

### How to test the changes in this Pull Request:

1. Go to the Activity page
2. Create a new activity with Media/Document/Video and content
3. Click on Media/Document/Video and can see the activity content

### Proof Screenshots or Video
https://vimeo.com/596598709/9cb3dc56fd


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
